extensions/kotlin/src/test/kotlin/io/opentelemetry/extension/kotlin/KotlinCoroutinesTest.kt

```diff
@@ -78,17 +78,19 @@ class KotlinCoroutinesTest {
   fun stressTest() {
     val context1 = Context.root().with(ANIMAL, "cat")
     runBlocking(context1.asContextElement()) {
+      assertThat(Context.current().get(ANIMAL)).isEqualTo("cat")
+      delay(10)
       assertThat(Context.current().get(ANIMAL)).isEqualTo("cat")
       for (i in 0 until 100) {
-        GlobalScope.launch {
```

Comment:
Not sure why I used GlobalScope here but it was wrong, it starts a job in a new context meaning

1) The ANIMAL is null, not cat
2) the result is not collected into `runBlocking` (so assertion failure was getting ignored)